docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb

```diff
@@ -370,14 +370,14 @@
       "-- Detecting CXX compile features - done\n",
```

Comment:
<p>legacy operator objects in <code>nvidia.dali.ops</code>  right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>